languages/python/exercises/concept/strings/.docs/instructions.md

```diff
@@ -0,0 +1,45 @@
+
+In this exercise you'll be processing log-lines.
+
+Each log line is a string formatted as follows: `"[<LEVEL>]: <MESSAGE>"`.
+
+There are three different log levels:
+
+- `INFO`
+- `WARNING`
+- `ERROR`
+
+You have three tasks, each of which will take a log line and ask you to do something with it.
+
+### 1. Extract a message from a log line
+
+Implement a function to return a log line's message:
+
+```python
+extract_message("[ERROR]: Invalid operation")
+// Returns: "Invalid operation"
+```
+
+The message should be trimmed of any whitespace.
+
+```python
+extract_message("[ERROR]: Invalid operation.\t\n")
+// Returns: "Invalid operation."
+
+### 2. Change a message's loglevel.
```

Comment:
I don't think this header is meant to be inside a triple-backtick escaped code block ... right now it looks like a comment (with too many hash marks) in a block of Python code.

Reply:
Indeed not, good catch.